test/extra/reportObject.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
       reportPrefix: 'FIREFOX_111.0.1__',
       assertionsCount: 1,
       lastError: null,
-      skipped: [
+      skippedAtRuntime: [
```

Comment:
if skipped is added back, why does the reportObject not have skipped anymore?

Reply:
Ahh missed it, but fixed now.